scripts/init-simapp-v2.sh

```diff
@@ -9,7 +9,7 @@ if [ -d "$SIMD_HOME" ]; then rm -rv $SIMD_HOME; fi
 $SIMD_BIN config set client chain-id simapp-v2-chain
 $SIMD_BIN config set client keyring-backend test
 $SIMD_BIN config set client keyring-default-keyname alice
-$SIMD_BIN config set app api.enable true
+$SIMD_BIN config set app rest-v2.enable true
```

Comment:
can we as well rename the server to simply `rest`? I don't think we should have v2 in there.